css/style.css

```diff
@@ -6,12 +6,197 @@ body {
     -webkit-font-smoothing: antialiased;
 }
 
+h1 {
+    padding-top: 17.5rem ;
+    font-size: 3.125rem;
+    color: white;
+    text-align: center;
+    font-weight: 300;
+}
+
+h2 {
+    padding-top: 1.25rem;
+    font-size: 1.563rem;
+    color: white;
+    text-align: center;
+    font-weight: 300;
+}
+
+
+.container {
+    width: 100%;
+    max-width: 900px;
+    margin: 0 auto;
+
+}
+
+/* The header section */
+.header {
+    width: 100%;
+    position: fixed;
+    top: 0;
+    background-color: white;
+    border-bottom: 1px solid #DDD;  
+}
+
+.header__inner {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    height: 70px;
+}
+
+
+
+/* nav */
+.nav {
+    font-size: 16px;
+
+}
+
+.nav a {
+    color: #5e5c5c;
+    display: inline-block;
+    vertical-align: top;
+    margin: 0 10px;
+}
+
+.nav a:hover {
+    color: #111111;
+
+}
+
+
+/* The hero section */
+.hero {
+    width: 100%;
+    height: 100vh;
+    /* The hero image */
+    background: url("../img/first-background.jpg") center no-repeat;
+    background-size: cover;
+
+}
+
+
+.hero__button {
+    width: 9.688rem;
+    background-color: #f15a29;
+    border-color: #f15a29;
+    text-shadow: none;
+    border-style: none;
+    border-radius: 4px;
+    padding: 1rem 1.5rem;
+    color: white;
+    font-size: 1.125rem;
+    margin: 25px auto;
+    display: block;
+}
+
+.hero__button:hover{
+    background-color: #D9400E;
+}
+
+
+
+/* main section */
+.main{
+    width: 100%;
+    height: 429px;
+    background-color: #fff;
+    
+}
+
+.main__title{
+    font-size: 2.75rem;
+    margin: 1.625rem 0 3.125rem;
+    font-weight: 300;
+    text-align: center;
+    color: #1d1e21;
+    margin-top: 120px;
+}
+
+
+.main__inner{
+    width: 900px;
+    margin:  auto;
+    display: inline-flex;
+    justify-content: space-evenly;
+    align-items: center;
+    
+
+}
+
+.main__block{
+    width: 20vw;
```

Comment:
Do you need a fixed width on these elements? Maybe some side padding would be enough and allow more flexibility.